runelite-client/src/main/java/net/runelite/client/plugins/pestcontrol/PestControlInfoOverlay.java

```diff
@@ -0,0 +1,164 @@
+ *     list of conditions and the following disclaimer.
+ *  2. Redistributions in binary form must reproduce the above copyright notice,
+ *     this list of conditions and the following disclaimer in the documentation
+ *     and/or other materials provided with the distribution.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ *  ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ *  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ *  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ *  ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ *  (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ *  ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ *  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ *  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.pestcontrol;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.inject.Inject;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Client;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.OverlayPriority;
+import net.runelite.client.ui.overlay.components.LineComponent;
+import net.runelite.client.ui.overlay.components.LineImageComponent;
+import net.runelite.client.ui.overlay.components.PanelComponent;
+import net.runelite.client.util.ImageUtil;
+import net.runelite.client.util.QueryRunner;
+
+@Slf4j
+public class PestControlInfoOverlay extends Overlay
+{
+	private final QueryRunner queryRunner;
+	private final Client client;
+
+	// Pest control game
+	@Getter(AccessLevel.PACKAGE)
+	private Game game;
+
+
+	private final PanelComponent panelComponent = new PanelComponent();
+
+	private static final Pattern PLAYERS_READY_PATTERN = Pattern.compile("<col=[a-z0-9A-Z]+>(.*)</col>");
+
+	private final BufferedImage hpIcon;
+	private final BufferedImage damageIcon;
+
+	@Inject
+	public PestControlInfoOverlay(QueryRunner queryRunner, Client client)
+	{
+		setPosition(OverlayPosition.TOP_LEFT);
+		setLayer(OverlayLayer.UNDER_WIDGETS);
+		setPriority(OverlayPriority.LOW);
+		this.queryRunner = queryRunner;
+		this.client = client;
+
+		this.hpIcon = ImageUtil.getResourceStreamFromClass(PestControlPlugin.class, "knight_hp_icon.png");
```

Comment:
Both of these icons should be available through `SpriteManager.getSprite`.

Reply:
They don't have spriteid's though. They have modelid's.

Reply:
That's fine then... In the future, we may want to consider adding some way to get an image from a model so we don't have to do this.